doc/source/dev_guide/index.rst

```diff
@@ -73,6 +73,16 @@ running::
 
     pytest satpy/tests
 
+Running benchmarks
+==================
+
+Satpy benchmarks are written using the
+`Airspeed Velocity <https://asv.readthedocs.io/en/stable/index.html>`_
+package (:mod:`asv`).
```

Comment:
Should you mention what dependencies are needed to run this?

Reply:
Or maybe mention what is generated and where/how to view them? Maybe a warning that these are fairly intensive tests (or could be) and aren't needed by the average developer?

Reply:
The dependencies are provided in the json file.